src/store/annotations/reducer.ts

```diff
@@ -1,6 +1,10 @@
 import { createReducer, combineReducers } from '@reduxjs/toolkit';
 import { AnnotationsState } from './types';
-import { createAnnotationAction } from './actions';
+import { createAnnotationAction, setActiveAnnotationAction } from './actions';
+
+const activeAnnotationId = createReducer<AnnotationsState['activeAnnotationId']>(null, builder =>
+    builder.addCase(setActiveAnnotationAction, (state, { payload: annotationId }) => annotationId),
```

Comment:
I'm wondering if this reducer should have some validation logic -- if the `annotationId` is not in the list of annotation ids in state, don't update the `activeAnnotationId`. Thoughts?

Reply:
I disagree. We should leave it as simple as possible until we see a need for more complex logic.